src/js/game/LevelMVC/LevelView.js

```diff
@@ -429,7 +431,8 @@ module.exports = class LevelView {
       "bubbleColumn": ["blocks", "Bubble_Column0", -12, 0],
       "conduit": ["blocks", "Conduit00", -12, 0],
 
-      "chest": ["blocks", "Chest0", -12, -20],
+      "Chest": ["blocks", "Chest0", -12, -20],
```

Comment:
Note this is a breaking change; Cole will need to update puzzles that use "chest" once this ships.

Reply:
👍 

I mean they should all get changed anyway to be <treasure>Chest right? No level has an empty chest? What a let down the students would find that! haha